src/services/InterviewService.js

```diff
@@ -767,6 +767,47 @@ async function updateCompletedInterviews () {
  */
 async function partiallyUpdateInterviewByWebhook (interviewId, webhookBody) {
   logger.info({ component: 'InterviewService', context: 'partiallyUpdateInterviewByWebhook', message: `Received webhook for interview id "${interviewId}": ${JSON.stringify(webhookBody)}` })
+
+  // this method is used by the Nylas webhooks, so use M2M user
+  const m2mUser = helper.getAuditM2Muser()
+  const bookingDetails = webhookBody.booking
+  const interviewStartTimeMoment = moment.unix(bookingDetails.start_time)
+  const interviewEndTimeMoment = moment.unix(bookingDetails.end_time)
+  let updatedInterview
+
+  if (bookingDetails.is_confirmed) {
+    try {
+      // CREATED + confirmed ==> inteview updated to scheduled
+      // UPDATED + cancelled ==> inteview expired
```

Comment:
This looks redundant, could we please remove it.

Reply:
Done in latest commit.